Example/ViewController.swift

```diff
@@ -600,6 +600,13 @@ extension ViewController {
             
             var coordinates: [CLLocationCoordinate2D] = [location.coordinate]
             trackPolyline?.appendCoordinates(&coordinates, count: UInt(coordinates.count))
+            
+            if mapView?.userTrackingMode != .followWithCourse, location.horizontalAccuracy < 65.0, location.speed > 8.0, let camera = mapView?.camera {
+                mapView?.userTrackingMode = .followWithCourse
```

Comment:
The example application doesn’t have a Recenter button, so this might be annoying to undo when stopping. Should we automatically change the camera back when at rest or add a button to change the tracking mode back?